xija/gui_fit.py

```diff
@@ -28,7 +28,10 @@
 from Chandra.Time import DateTime
 import pyyaks.context as pyc
 
-import Chandra.taco
+try:
+    import acis_taco as taco
```

Comment:
I think this is fine, but I am a little confused by the application.  To actually execute this code in gui_fit, would we need to backport/install in the 0.17 gtk skare?

Reply:
@jeanconn I suspect it's not necessary to backport--the reproducibility aspect of the random numbers is actually turned off in `gui_fit`, so that sort of obviates the main purpose of the new code. 

ACIS ops personnel still use `gui_fit` fairly regularly. I have a Bokeh-based version I'm developing for `xijafit`, but it will take a while before it is ready.

Reply:
OK.  I can confirm that if we were to run this new xija 3.10 in 0.17 skare, the import looks like it would correctly pick up Chandra.taco .  Not sure about the best ways to run model tests these days to quickly confirm the xija.component.heat bits .

``
In [2]: xija.gui_fit.taco
Out[2]: <module 'Chandra.taco' from '/proj/sot/ska/0.17/arch/x86_64-linux_CentOS-5/lib/python2.7/site-packages/Chandra.taco-0.2-py2.7.egg/Chandra/taco/__init__.pyc'>
In [4]: os.environ['SKA']
Out[4]: '/proj/sot/ska/0.17'
In [6]: xija.__version__
Out[6]: '3.10'
```